src/redist/targets/GenerateDebs.targets

```diff
@@ -276,30 +299,29 @@
 
   <Target Name="TestSdkDeb"
       Condition=" '$(CLIBUILD_SKIP_TESTS)' != 'true' and '$(IsDebianBaseDistro)' == 'True' and '$(DebuildPresent)'  == 'true' "
-      DependsOnTargets="RestoreTests;GetAspNetSharedFxInstallArgs"
-      Inputs="$(DownloadedSharedHostInstallerFile);
+      DependsOnTargets="GetAspNetSharedFxInstallArgs"
+      Inputs="$(DownloadedRuntimeDepsInstallerFile);
+              $(DownloadedNetCoreAppTargetingPackInstallerFile);
+              $(DownloadedAspNetTargetingPackInstallerFile);
+              $(DownloadedSharedHostInstallerFile);
               $(DownloadedHostFxrInstallerFile);
               $(DownloadedSharedFrameworkInstallerFile);
-              $(SdkDebInstallerFile);"
+              $(DownloadedAspNetCoreSharedFxInstallerFile);
+              $(SdkDebInstallerFile);
+              $(DebianTestResultsXmlFile);"
       Outputs="$(DebianTestResultsXmlFile)" >
 
       <!-- Install Dependencies and SDK Packages -->
       <Exec Command="sudo dpkg -i $(DownloadedRuntimeDepsInstallerFile)" />
+      <Exec Command="sudo dpkg -i $(DownloadedNetCoreAppTargetingPackInstallerFile)" />
+      <Exec Command="sudo dpkg -i $(DownloadedAspNetTargetingPackInstallerFile)" />
       <Exec Command="sudo dpkg -i $(DownloadedSharedHostInstallerFile)" />
       <Exec Command="sudo dpkg -i $(DownloadedHostFxrInstallerFile)" />
       <Exec Command="sudo dpkg -i $(DownloadedSharedFrameworkInstallerFile)" />
       <Exec Command="sudo dpkg $(InstallAspNetCoreSharedFxArgs) -i $(DownloadedAspNetCoreSharedFxInstallerFile)" />
 
       <Exec Command="sudo dpkg -i $(SdkDebInstallerFile)" />
 
-      <!-- Run E2E -->
```

Comment:
I think we should keep an issue referenced for enabling end-to-end against bits coming out of the installer (instead of bits going into it from layout). This is coverage we also lost. But since this change is making more things not less things run, it does not have to be adressed in the is PR, but I don't want to lose track of it, either.